mantra-game/plugins/gamepad/Gamepad.js

```diff
@@ -81,49 +142,55 @@ export default class Gamepad {
       }
     }
   }
+  
+  controllerConfig(hash) {
+    const config = typeof this.configs[hash] === 'object'?
+      this.configs[hash]:
+      Gamepad.defaultControllerConfig;
+    if(!this.configs[hash]){
+      this.configs[hash] = true; //don't double load
+      (async ()=>{
+        let loadedConfig = null;
+        try{
+          loadedConfig = (await import(`./gamepads/${hash.slice(0)}.js`)).default;
```

Comment:
dynamic imports are currently not allowed inside mantra. This is a current requirement of the build system and plugin system. It should be possible to enable dynamic imports for the entire project; we need to add additional test coverage for `Game.use()` and most likely adjust the current build system.